templates/base.html

```diff
@@ -126,8 +126,9 @@
             You can contribute to, or report problems with,
             <a class="p-link--external" href="https://bugs.launchpad.net/snapd">snapd</a>,
             <a class="p-link--external" href="https://bugs.launchpad.net/snapcraft">Snapcraft</a>,
+            <a class="p-link--external" href="https://forum.snapcraft.io/c/doc">documentation content</a>
             or
-            <a class="p-link--external" href="https://github.com/canonical-websites/snapcraft.io/issues">this site</a>.
+            <a class="p-link--external" href="https://github.com/canonical-websites/docs.snapcraft.io/issues">this website application</a>.
```

Comment:
`website application` sounds weird.
I understand that it's to explain that this has dynamic content but it sounds weird.

Reply:
Any suggestions?

Reply:
why not keeping `this site` ?

Reply:
Yes okay that makes sense. I was trying to extra discourage people from filing bugs about incorrect content, but I guess maybe just saying "documentation content" is enough. At least we can leave it how you suggest and see if we get many people filing issues when they shouldn't.

Reply:
Done